### DISCUSSION TITLE
Getting error page on refresh of website

### DISCUSSION THREAD

**Discussion by hayleyiscoding** [vercel/community#2219] [Help]
### Summary

I adding a custom domain to my website from GoDaddy. My website works with this custom domain, but if I refresh the website, it goes to an error page. Can anyone help? Thanks.

**Comment by amyegan** [comment]
Hi @hayleyiscoding! Can you tell me more about how you added the domain (i.e. A record, CNAME record, nameserver change)? If you are able to share the domain, I can take a closer look at what is going wrong.
